### DISCUSSION TITLE
Add Dusk Selector to Checkbox

### DISCUSSION THREAD

**Discussion by StanBarrows** [laravel/nova-issues#4929] [Q&A] [answered]
- [ ] Currently checkboxes do not have a dusk selector. 

`<input type="checkbox" class="checkbox mt-2" id="published-update-auftragstyp-bestellung-boolean-field" name="Veröffentlicht">`

<img width="1193" alt="grafik" src="https://user-images.githubusercontent.com/10268813/196020589-7366334e-605c-44ea-b5af-0d8a3fd42eb0.png">

**Comment by crynobone** [comment, marked as answer]
You can use the name attribute.
